internal/databases/greenplum/backup_push_handler.go

```diff
@@ -204,16 +204,12 @@ func (bh *BackupHandler) HandleBackupPush() {
 		bh.abortBackup()
 	}
 
-	// WAL-G will reconnect later
```

Comment:
Previously there were problems with keeping the master connection alive while taking long backups.

Reply:
Checked it on 5 hour and 12 hour backups, seems okay, will continue to monitor it